src/ui/handlers/achievements-ui-handler.ts

```diff
@@ -11,14 +11,17 @@ import type { Voucher } from "#system/voucher";
 import { getVoucherTypeIcon, getVoucherTypeName, vouchers } from "#system/voucher";
 import { MessageUiHandler } from "#ui/message-ui-handler";
 import { ScrollBar } from "#ui/scroll-bar";
+import { ScrollableGridController } from "#ui/scrollable-grid-controller";
 import { addTextObject } from "#ui/text-utils";
 import { addWindow } from "#ui/ui-theme";
 import i18next from "i18next";
 
-enum Page {
-  ACHIEVEMENTS,
-  VOUCHERS,
-}
+const Page = {
+  ACHIEVEMENTS: 0,
+  VOUCHERS: 1,
```

Comment:
I'd like all the enums to start at `1` (unless there is an explicit "none" value where `enum.none` is equivalent to it being `nil`).
```suggestion
  ACHIEVEMENTS: 1,
  VOUCHERS: 2,
```